app/services/request_create_service.rb

```diff
@@ -51,15 +51,9 @@ def create_child_requests(request)
   end
 
   def update_leaf_with_workflow(leaf_request, workflow_id, group_ref)
-    group_name = if group_ref
-                   ContextService.new(leaf_request.context).as_org_admin do
-                     Group.find(group_ref).name
-                   end
-                 else
-                   'System approval'
-                 end
-
-    leaf_request.update!(:workflow_id => workflow_id, :group_ref => group_ref, :group_name => group_name)
+    group_name = group_ref.try(:[], 'name') || 'System approval'
```

Comment:
Recommed making `System approval` a constant in this class and update the spec to reference the constant as well.  https://github.com/RedHatInsights/approval-api/blob/master/spec/services/request_create_service_spec.rb#L182